src/mpid/ch4/netmod/ofi/ofi_rma.c

```diff
@@ -66,7 +66,9 @@ int MPIDI_OFI_nopack_putget(const void *origin_addr, MPI_Aint origin_count,
     /* allocate request */
     MPIDI_OFI_win_request_t *req = MPIDI_OFI_win_request_create();
     MPIR_ERR_CHKANDSTMT((req) == NULL, mpi_errno, MPIX_ERR_NOREQ, goto fn_fail, "**nomemreq");
-    req->vni = MPIDI_WIN(win, am_vci);
+    req->vni_local = MPIDI_WIN(win, am_vci);
+    req->vni_target = MPIDI_WIN_TARGET_VCI(win, target_rank);
```

Comment:
Would it be make sense to change the name `MPIDI_WIN` to something similar as `MPIDI_WIN_TARGET_VCI`?

Reply:
`MPIDI_WIN` is a macro to access the `win` struct. But I guess we can wrap into a new macro, `MPIDI_WIN_LOCAL_VCI`. Let's do that separately.

Reply:
That sounds good